db/re/item_db_usable.yml

```diff
@@ -15484,11 +15484,8 @@ Body:
   - Id: 12999
     AegisName: Branch_Of_Dead_Tree_Box
     Name: Dead Branch Box
-    Type: Cash
```

Comment:
I believe `Type:` is required for all items, the next most suitable type we could use is `Usable` which avoids the confirmation dialogue